### DISCUSSION TITLE
Will the repository pattern be implementable with prism?

### DISCUSSION THREAD

**Discussion by aldodarrigos** [prisma/prisma#18331] [Q&A] [answered]
Hello community, I have seen some discussions on this topic but I think they never reached a definitive solution.
I wanted to know if as of today they have already taken any action on this issue?
Thanks for your help.

**Comment by ludralph** [comment, marked as answer]
Hi @aldodarrigos 👋 

We currently don't have a concrete ETA for this topic. You can, however, follow the [roadmap](https://www.notion.so/prismaio/Prisma-Roadmap-50766227b779464ab98899accb98295f?p=525cf29) to see what we are actively working on.

**Reply by nurul3101** [reply]
For reference, here is a GitHub Discussion regarding repository pattern that you can refer
- https://github.com/prisma/prisma/discussions/3929

**Reply by nurul3101** [reply]
@aldodarrigos Can you have a look at the Discussion? Please let us know in case you have any other queries.

**Reply by nurul3101** [reply]
Please feel free to let us know in case you have any additional queries.
